k8_cortx_cloud/destroy-cortx-cloud.sh

```diff
@@ -460,9 +432,9 @@ function deleteKubernetesPrereqs()
 function deleteCortxNamespace()
 {
     # Delete CORTX namespace
-    if [[ "${namespace}" != "default" ]]; then
-        uninstallHelmChart "cortx-ns-${namespace}"
-    fi
+    #if [[ "${namespace}" != "default" ]]; then
```

Comment:
I'm not sure about this... I need to review this and consider it more carefully.

Reply:
I restored this one.